src/Parser.php

```diff
@@ -307,6 +313,9 @@ public function addImportedSymbol(string $type, string $alias, ?string $name = n
         $this->importedSymbols[0][$type][$alias] = ['name' => $name, 'node' => $internalRef];
     }
 
+    /**
+     * @return string|null
```

Comment:
This is wrong. It returns `array{name: string, node: AssignTemplateVariable|null}`